### PR TITLE
Cert-manager now uses pinned version

### PR DESCRIPTION
**Pull request by Waterdrips**
## Description

The cert-manager app was using a string rather than the
version variable when the chart was installed. This moves
it to using the version variable

Signed-off-by: Alistair Hey <alistair.hey@form3.tech>

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
https://github.com/alexellis/arkade/commit/3b6845bced66cecfd302f153792ae10052718596#r40689484
`
## How Has This Been Tested?

installed on amd64 k3d cluster
cert-manager installed and ran correctly

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [ ] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: cmd/apps/certmanager_app.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ func MakeInstallCertManager() *cobra.Command {
 	certManager.RunE = func(command *cobra.Command, args []string) error {
 		wait, _ := command.Flags().GetBool("wait")
 		const certManagerVersion = "v0.12.0"
```
Comment (alexellis) [inline cmd/apps/certmanager_app.go:36]: Are these read from flags now?

Comment (meyskens) [reply cmd/apps/certmanager_app.go:36]: if it does it will error on any release after 0.14 as the CRDs moved

Comment (Waterdrips) [reply cmd/apps/certmanager_app.go:36]: its currently only pinned to 0.12, the pr from @meyskens to move to 0.15 allows version flag.

**Comment by Waterdrips** [comment]
/close 
this was superseded by #154

**Pull request closed without merging**